src/cellularnoise.rs

```diff
@@ -15,76 +16,75 @@ fn noise_gen(
     height_as_str: &str,
 ) -> Result<String> {
     let percentage = percentage_as_str.parse::<i32>()?;
```

Comment:
why not usize this?

Reply:
didn't want to touch the API of this, could reasonably change this to usize.